Api/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.9"
+
+services:
+  postgres:
```

Comment:
You are missing the creation of tables using an SQL script when the container is started

Reply:
Done